sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncContainer.java

```diff
@@ -1442,33 +1446,152 @@ ThroughputControlGroup enableThroughputLocalControlGroup(String groupName, int t
      * @return A {@link ThroughputControlGroup}.
      */
     @Beta(value = Beta.SinceVersion.V4_13_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
-    ThroughputControlGroup enableThroughputLocalControlGroup(String groupName, double targetThroughputThreshold) {
-        return this.enableThroughputLocalControlGroup(groupName, targetThroughputThreshold, false);
+    public ThroughputControlGroup enableThroughputLocalControlGroup(String groupName, double targetThroughputThreshold) {
+        return this.enableThroughputControlGroup(
+            ThroughputControlGroupFactory.createThroughputLocalControlGroup(groupName, this, targetThroughputThreshold));
     }
 
     /**
      *
      * @param groupName The throughput control group name.
      * @param targetThroughputThreshold The target throughput threshold for the control group.
      * @param isDefault Flag to indicate whether this group will be used as default.
+     *
+     * @return A {@link ThroughputControlGroup}.
+     */
+    @Beta(value = Beta.SinceVersion.V4_13_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
+    public ThroughputControlGroup enableThroughputLocalControlGroup(String groupName, double targetThroughputThreshold, boolean isDefault) {
+        return this.enableThroughputControlGroup(
+            ThroughputControlGroupFactory.createThroughputLocalControlGroup(groupName, this, targetThroughputThreshold, isDefault));
+    }
+
+    //endregion
+
+    // region enableThroughputGlobalControlGroup
+
+    /**
+     *
+     * @param groupName The throughput control group name.
+     * @param targetThroughput The target throughput for the control group.
+     * @param globalControlContainer The global control container. This is the container which will be used to track the traffic load of each client.
+     * @param controlItemRenewInterval The global control container item renew interval.
```

Comment:
NIT: Brief explanation what renewal and expire mean - or if you checked it in already a link to your design document

Reply:
updated